nav2_bringup/bringup/params/nav2_params_voxel.yaml

```diff
@@ -0,0 +1,192 @@
+amcl:
```

Comment:
This doesn’t need to be a new param file, just add the voxel layer to the existing yaml

Reply:
Yeah, it should just be one additional entry into the costmap layers name & plugin type vectors and the configuration thereafter